resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/kubernetes/InitContainerResourceStagingServerSecretPluginSuite.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.deploy.kubernetes
+
+import org.scalatest.BeforeAndAfter
+import io.fabric8.kubernetes.api.model._
+import org.apache.spark.deploy.kubernetes.constants._
+
+import scala.collection.JavaConverters._
+import org.apache.spark.SparkFunSuite
+
+class InitContainerResourceStagingServerSecretPluginSuite extends SparkFunSuite with BeforeAndAfter{
+  private val INIT_CONTAINER_SECRET_NAME = "init-secret"
+  private val INIT_CONTAINER_SECRET_MOUNT = "/tmp/secret"
+
+  private val initContainerRSSP = new InitContainerResourceStagingServerSecretPluginImpl(
+    INIT_CONTAINER_SECRET_NAME,
+    INIT_CONTAINER_SECRET_MOUNT
+  )
+
+  test("Volume Mount into InitContainer") {
+    val returnedCont = initContainerRSSP.mountResourceStagingServerSecretIntoInitContainer(
+      new ContainerBuilder().withName("init-container").build()
+    )
```

Comment:
This closing bracket should be on the previous line. There's a few places where this is done, so please fix the others also.